app/views/attachments/MultipleDocumentsRequired.scala.html

```diff
@@ -46,7 +47,15 @@
             case VAT5L => vat5LBlock
             case TaxRepresentativeAuthorisation => vat1TRBlock
             case attachment => Html(messages(s"attachments.multipleDocumentsRequired.${AttachmentType.map(attachment)}"))
-        }: _*
+        }
+        ::: attachments.map{
```

Comment:
You could filter for VAT5L here first and then map. That'd just look simpler imho. Same in the email and postal cover sheets.